Source/EasyNetQ/ReceiveConfiguration.cs

```diff
@@ -179,6 +187,12 @@ public IReceiveConfiguration WithQueueMode(string queueMode)
         return this;
     }
 
+    public IReceiveConfiguration WithQueueType(string queueType)
```

Comment:
```suggestion
    public IReceiveConfiguration WithQueueType(string queueType = QueueType.Classic)
```

to align with `ISubscriptionConfiguration WithQueueType(string queueType = EasyNetQ.QueueType.Classic)`